plugins/deprecation.nix

```
@@ -1,5 +1,13 @@
 { lib, ... }:
 let
+  deprecated = {
+    # TODO: added 10-23-2024, move to removed after 24.11
+    "rust-tools" = ''
+      The `rust-tools` project has been abandoned.
+
+      It is recommended to use `rustaceanvim` instead.
```

Since this is stdout and not markdown, I don't think we need the blankline?
```suggestion
      The `rust-tools` project has been abandoned.
      It is recommended to use `rustaceanvim` instead.
```
Or, if you do want it, perhaps we should consider also having one when joining the generic warning with this message?